### ISSUE TITLE
Duplicate results

### ISSUE THREAD

**Issue by alongreyber** [ranking-agent/strider#61]
The following query should work:
```json
{
    "message": {
        "query_graph": {
            "nodes": {
                "a": {
                    "id": "CHEBI:89564",
                    "category": "biolink:ChemicalSubstance"
                },
                "b": {
                    "category": "biolink:Disease"
                }
            },
            "edges": {
                "ab": {
                    "subject": "a",
                    "object": "b",
                    "predicate": "biolink:related_to"
                }
            }
        }
    }
}
```

**Comment by patrickkwang** [comment]
This returns an answer as of #83, but it's actually duplicated 12 times... I'm repurposing this issue to address that. I suspect Strider and Plater are both traversing the Biolink hierarchy, and Strider is not properly deduplicating at the end.

**Comment by alongreyber** [comment]
Fixed by #89.
